dracut/modules.d/90kiwi-live/kiwi-live-genrules.sh

```diff
@@ -15,7 +15,11 @@ case "${root}" in
         wait_for_dev -n "${root#live:}"
         ;;
     live:aoe:/dev/*)
-        /sbin/initqueue \
-            --settled --onetime --unique /sbin/kiwi-live-root "${root}"
+        {
+        printf 'KERNEL=="%s", RUN+="/sbin/initqueue %s %s %s"\n' \
+            "${root#live:aoe:/dev/}" "--settled --onetime --unique" \
+            "/sbin/kiwi-live-root" "${root#live:aoe:}"
+        } >> /etc/udev/rules.d/99-live-aoe-kiwi.rules
+        wait_for_dev -n "${root#live:aoe:}"
```

Comment:
I like this :+1: Synchronizing the execution of the script in initqueue based on udev events looks like a rubost a approach to me.

Reply:
Thanks and seems to be approved by the great people testing :)